llvm/test/CodeGen/PowerPC/subreg-coalescer.mir

```diff
@@ -0,0 +1,27 @@
+# RUN: llc -mtriple powerpc64le-unknown-linux-gnu -mcpu=pwr8 -x mir < %s \
```

Comment:
Can we pre-commit this test case? Without this PR, this case should fail verification?

Reply:
> Can we pre-commit this test case? Without this PR, this case should fail verification?

Well, the issue is that without the fix in `RegisterCoalescer.cpp` the test will fail. However, without this entire patch this test will still pass. We need the fix in the register classes for this test to start failing.

Reply:
Ok, I see. The `HasDisjunctSubRegs` bit is false without this PR, so that `MRI->shouldTrackSubRegLiveness(...)` returns false.